packages/libp2p/src/connection-manager/dial-queue.ts

```diff
@@ -230,9 +232,22 @@ export class DialQueue {
         // clean up abort signals/controllers
         signal.clear()
       })
-      .catch(err => {
+      .catch(async err => {
```

Comment:
if i'm not mistaken, this dial-queue is also used for direct dials, should we discern between an auto-dial triggered dial vs a manually/user requested dial?